File: 이지민/demo/src/main/java/com/todo/demo/dto/user/UserDTO.java
package com.todo.demo.dto.user;

import com.todo.demo.model.User;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

public class UserDTO {

    @Getter
    public static class Response{
        private Long id;
        private String email;
        private String username;

    }
}

File: 박진용/todo/src/user/dto/user.dto.ts
import { addUserDto } from './adduser.dto';
export class User {
  public constructor(addUserRequest: addUserDto) {
    this.email = addUserRequest.email;
    this.username = addUserRequest.username;
    this.profile_image = addUserRequest.profile_image;
  }

  email: string;
  password: string;
  username: string;
  profile_image: string;
}
